### ISSUE TITLE
Same model for classification as well as generating samples

### ISSUE THREAD

**Issue by Ektagavas** [ermongroup/mintnet#3]
Hey, thank you for sharing the code. It's great!
I had one query. I have trained a Mintnet model for classification on CIFAR10. Can I use this same model for generating new samples i.e as a generative model (with some code modifications, of course)? If yes, can you suggest what part of the code needs to be changed?
As far as I understood, currently, density estimation and classification experiments are run independently, but I want to make changes to achieve the above use-case. 
Thanks in advance!

**Comment by yang-song** [comment]
Hello, I don't think it's easy to convert a classifier to a generative model. For image generation, you will need to first have the logits layer of a MintNet classifier and then compute the inverse. The first step is non-trivial.
